zero/colossalai_utils/gpt2_config.json

```diff
@@ -1,11 +1,12 @@
 {
     "method": "colossalai",
     "model": {
-        "type": "gpt2_10b"
+        "type": "gpt2_10b",
+        "numel": 9.57
```

Comment:
? why hard code?

Reply:
It takes too long to build the model. We want to save some time for now.

Reply:
Initialization time is too long.

Reply:
This is suitable for all methods (Fairscale / Deepspeed / Ours).